bin/node-template/runtime/src/lib.rs

```diff
@@ -336,7 +336,7 @@ impl_runtime_apis! {
 
 	impl sp_api::Metadata<Block> for Runtime {
 		fn metadata() -> OpaqueMetadata {
-			Runtime::metadata().into()
+			OpaqueMetadata::new(Runtime::metadata().into())
```

Comment:
It would be good to keep `frame-metadata` as lightweight as possible and not coupled to any substrate dependencies, so I thing the `opaque_metadata` function is a better idea.

Reply:
Added a follow up for this in the description